app/models/service_template_ansible_tower.rb

```diff
@@ -52,6 +52,10 @@ def self.validate_config_info(options)
     config_info
   end
 
+  def my_zone
+    job_template.manager.try(:my_zone) || MiqServer.my_zone
```

Comment:
@gmcculloug If we return nil instead of defaulting to MiqServer.my_zone, the work is put on the queue with a nil zone. I initially thought the service_template_provision_task deliver_to_automate method would catch it, but it only defaults it if the source doesn't respond to my_zone.

determine_queue_zone will leave it as nil because the options key zone exists:
`if options.key?(:zone)
   options[:zone] # return specified zone including nil (aka ANY zone)`